version/1/4/mk/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# Пакт за Придонесување - Kодекс на Oднесување
+
+## Нашиот завет
+
+Во интерес на негувањето на отворена и пријатна околина, ние како придонесувачи и одржувачи ветуваме дека ќе го правиме учеството во нашиот проект и нашата заедница пријатно искуство за сите, без вознемирување, без разлика на возраст, големина на телото, инвалидитет, етничка група, родов идентитет и претставување, ниво на искуство, националност, личен изглед, раса, религија, или сексуален идентитет и ориентација.
+
+## Нашите стандарди
+
+Примерите за однесување кое придонесува за позитивна околина вклучуваат:
```

Comment:
I had an error there, using "Примерите" instead of "Примери", I've edited my previous comment once I realized that. My intention with this change was that the whole sentence to be in plural. It actually should be: "Примери за однесувања кои придонесуваат за позитивна околина:" or "Примери на однесувања кои придонесуваат за позитивна околина:". What do you think?